src/units/PhysicalConstexprs.hh

```diff
@@ -248,6 +253,9 @@ private:
   //! Electron volt
   static constexpr double d_electronVolt_SI = d_electronCharge_SI; // J
 
+  //! Atomic mass unit (CODATA 2010 value)
```

Comment:
Let me know if you are interested in moving our code to use `CODATA 2018` data.  This will require some coordination with the bigger code teams.

Reply:
Not a pressing concern for me now, I'm happy to accommodate whatever is convenient for working with the bigger teams.